src/headerr/Header.tsx

```diff
@@ -1,40 +1,46 @@
 import { Link } from "react-router-dom";
-import { AppBar, Toolbar, Typography, Button, Box } from "@mui/material";
+import { AppBar, Toolbar, Button, Box } from "@mui/material";
 import { useMetaMask } from "../contexts/MetaMaskContext";
+import styled from "styled-components";
 
-const Header = () => {
-  const { isMetamaskConnected, connectToMetamask, disconnectFromMetamask } =
-    useMetaMask();
+const StyledAppBar = styled(AppBar)`
+  && {
+    background: linear-gradient(
+      109.6deg,
+      rgb(9, 9, 121) 11.2%,
+      rgb(144, 6, 161) 53.7%,
+      rgb(0, 212, 255) 100.2%
+    );
+    height: 7vh;
```

Comment:
why do you use vh here?

Reply:
Actually, this line is not needed at all, i will remove it